spec/corrector_spec.rb

```diff
@@ -0,0 +1,7 @@
+require './corrector'
+describe Corrector do
+  it 'Add capitalized first lettre using correct_name method' do
+    corrector = Corrector.new
+    expect(corrector.correct_name('taher')).to eql('Taher')
```

Comment:
- [ ] please make sure to be thorough when testing, the method in this class is also responsible for making sure the name is not longer than 10 characters, please make sure to test that as well.